static/src/stylesheets/module/football/_overview.scss

```diff
@@ -1,3 +1,160 @@
+/* ==========================================================================
+   World Cup 2018
+   ========================================================================== */
+.wc-overview {
+    .container__title {
+        &:after {
+            display: none;
+        }
+    }
+}
+
+.facia-header {
+    .facia-header__inner {
+        padding-top: $gs-baseline / 2;
+        background-color: $news-garnett-highlight;
+        padding-bottom: 0px;
```

Comment:
Let's use `0` instead of `0px`, fewer bytes!

Reply:
done.